pkg/orchestrator/etcd_worker.go

```diff
@@ -283,14 +283,33 @@ func (worker *EtcdWorker) cloneRawState() map[util.EtcdKey][]byte {
 	return ret
 }
 
+var maxBatchPatchSize = 96
+
 func (worker *EtcdWorker) applyPatchGroups(ctx context.Context, patchGroups [][]DataPatch) ([][]DataPatch, error) {
+	getBatchPatches := func(patchGroups [][]DataPatch) ([]DataPatch, int) {
+		batchPatches := patchGroups[0]
+		patchesNum := 1
+		for _, patches := range patchGroups {
+			if patchesNum == 1 {
+				continue
+			}
+			if (len(batchPatches) + len(patches)) > maxBatchPatchSize {
```

Comment:
what if patchGroups[0] already > max, need break down or just leave as it is.

Reply:
Leave as it is.